theme/themes/eea/globals/site.overrides

```diff
@@ -101,3 +101,17 @@ a {
   background: @lightLavender;
   color: @black;
 }
+
+/* override slate-metadata addon styles */
+
+ .title.metadata.mention,
+ .description.metadata.mention,
+ .datetime.metadata.mention,
+ .text.metadata.mention,
+ .other_organisations.metadata.mention,
+ .preview_caption.text.metadata.mention,
+ .publisher.metadata.mention,
+ .rights.metadata.mention,
+ .topics.text.metadata.mention {
+   all: unset !important;
```

Comment:
@ichim-david This reverts all styles applied on slate-metadata block, you think this should be the part of policy addon?

Reply:
@nileshgulia1  yes I don't like this nuke all option to have in the design package.
It seems that this is better suited for the individual site.
I can accept the padding and margin with variables but not the all unset.